### DISCUSSION TITLE
Let an asset be imported from an ID that Craft export provides

### DISCUSSION THREAD

**Discussion by texasbaptists** [engram-design/FeedMe#1014] [Ideas] [answered]
I need to migrate entries from an old section to a new one. When entries with an asset field (in my case a featured image) are exported from Craft, the JSON data gives an asset ID. It would be super helpful if I could import this JSON into the new section and the asset field be populated with the assets that are already in the system by ID. 

Here's an example of how an asset field shows up when exporting craft entries:

```
        "featuredImage": [
            "168975"
        ],
```

**Comment by cole007** [comment]
100% could do with this feature

**Comment by johnj24** [comment]
I could use this feature as well. Element Exporter and Feed Me do not feel 100% compatible otherwise. I posed this question in April 2020 and did get one response recently if it helps anyone: https://craftcms.stackexchange.com/questions/35215/importing-assets-with-feed-me-using-element-exporter-data

**Comment by davidhellmann** [comment]
Any updates here. Have to move entries to products and it would be nice to use the core exports for it.

**Comment by valboivin** [comment]
I'm also trying to figure out if there's a way of doing this! We have to refactor our entire website to a new structure. Would be truly helpful to be able of linking already existing assets or entries with ID

**Comment by angrybrad** [comment, marked as answer]
Resolved in https://github.com/craftcms/feed-me/pull/1327 and will be included in the next release.
